bode/bode/resources/tasks/schemas.py

```diff
@@ -4,7 +4,12 @@
 from bode.models.enums import DirectedRelationType, TaskStatus
 from bode.models.task_relation.actions import is_task_blocked
 from bode.resources.base_schema import BaseSchema
-from bode.resources.tags.schemas import TagInputSchema, TagSchema
+from bode.resources.tags.schemas import TagSchema
+
+
+class RelationInputSchema(BaseSchema):
+    task_id = fields.UUID(required=True)
+    type = fields.String(validate=validate.OneOf(DirectedRelationType.list()), required=True)
```

Comment:
You're not handling the subtask relation types so they should be excluded from the possible types here